### ISSUE TITLE
missing Schema function (?) when running generateCore

### ISSUE THREAD

**Issue by stebick** [NeurodataWithoutBorders/matnwb#30]
Hi matnwb experts, 

sorry if this is a naive question. 

I tried importing a NWB dataset to matlab for the first time. I ran the command "generateCore" and received an error (pasted below). I also tried to run it using the nwb.namespace.yaml from pynwb. 

Am I missing a Schema function? 

I ran it on Matlab R2017b and using matnwb code updated 6/6/18 from github. 

Thank you, 
Stephan 

generateCore('/pathto/matnwb/schema/core/nwb.namespace.yaml');

Warning: Invalid file or directory '/pathto/pynwb/src/pynwb/jar/schema.jar'. 
> In javaclasspath>local_validate_dynamic_path (line 271)
  In javaclasspath>local_javapath (line 187)
  In javaclasspath (line 124)
  In javaaddpath (line 71)
  In util.generateSchema (line 2)
  In generateCore (line 22) 
Undefined function or variable 'Schema'.

Error in yaml.getSourceInfo (line 2)
schema = Schema();

Error in util.generateSchema (line 5)
schema = yaml.getSourceInfo(localpath, filenames{:});

Error in generateCore (line 22)
cs = util.generateSchema(core);

**Comment by ln-vidrio** [comment]
util.generateSchema assumed that the working directory is in the matnwb directory, so the constructed filepath to jar/schema.jar ends up being incorrect.

Quickest workaround without pulling is to move MATLAB's current folder to the matnwb root but it's also a two-line fix:

```MATLAB
function namespace = generateSchema(source)
%find jar from source
matnwbDir = fileparts(which('nwbfile'));
javaaddpath(fullfile(matnwbDir, 'jar', 'schema.jar'));
```

**Comment by ln-vidrio** [comment]
Should be fixed now ecb6f03
